README.md

```diff
@@ -62,9 +62,14 @@ Use "gitops [command] --help" for more information about a command.
 
 For more information please see the [docs](https://docs.gitops.weave.works/docs/cli-reference)
 
-## CLI/API development
+## Development
 
-To set up a development environment for the CLI
+### To use a bootstrapped, ready made environment
```

Comment:
🤔  I wonder if we can clarify somehow that the `ready made environment` means to have a kind cluster created by the command `KIND_CLUSTER_NAME=<some name> ./tools/kind-with-registry.sh`. Otherwise `make cluster-dev` will fail. Perhaps we need to switch the order of the blocks `To use an existing environment`(which actually seem like it needs to change to something like `creating a dev env from scratch` or something similar) and `To use a bootstrapped, ready made environment`. 🙂

Reply:
I think let's remove this thing. People can use the other section and figure out that they can leave their cluster for reuse if they want to. This section has always really confused me to the point that I pretend it isn't there